bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/Canopus.java

```diff
@@ -227,154 +152,99 @@ public GuideStarValidation validate(SPTarget guideStar, ObsContext ctx) {
+                                    .map(base -> {
+                                        final CoordinateDiff diff = new CoordinateDiff(base, gscoords);
+
+                                        // Get offset and switch it to be defined in the same coordinate
+                                        // system as the shape.
+                                        final Offset dis = diff.getOffset();
+                                        final double p = -dis.p().toArcsecs().getMagnitude();
+                                        final double q = -dis.q().toArcsecs().getMagnitude();
+                                        final Set<Offset> offsets = new TreeSet<>();
+                                        offsets.add(offset);
+
+                                        final Area a = Canopus.instance.offsetIntersection(ctx, offsets);
+                                        return a != null && a.contains(p, q);
+                                    })))
+                    .getOrElse(false);
         }
     }
 
-        //     REL-1042: Update the Canopus probe limits in the OT
-//    public static final double RADIUS_ARCSEC = 60.0;
     public static final double RADIUS_ARCSEC = 62.5;
 
-
-    // Probe ranges expressed in arcsec
-//    private static final Tuple2<Integer, Integer> PROBE1_RANGE = new Pair<Integer, Integer>(-66, 24);
-//    private static final Tuple2<Integer, Integer> PROBE2_RANGE = new Pair<Integer, Integer>(-24, 66);
-
-    // REL-297:
     // P1 can access a square field of 3.7 x 3.0 armcin, but the center is at (+0.11,+0.3)
     // P2 can access a square field of 4.2 x 3.0 arcmin, but the center is at (0.0,-0.33)
     private static final Point2D.Double PROBE1_CENTER = new Point2D.Double(0.11 * 60.0, 0.3 * 60.0);
     private static final Point2D.Double PROBE2_CENTER = new Point2D.Double(0.0, -0.33 * 60.0);
     private static final Tuple2<Double, Double> PROBE1_DIM = new Pair<>(3.7 * 60.0, 3.0 * 60.0);
-
-        //     REL-1042: Update the Canopus probe limits in the OT
-//    private static final Tuple2<Double, Double> PROBE2_DIM = new Pair<Double, Double>(4.2 * 60.0, 3.0 * 60.0);
     private static final Tuple2<Double, Double> PROBE2_DIM = new Pair<>(4.2 * 60.0, 2.5 * 60.0);
-
     private static final Ellipse2D AO_PORT = new Ellipse2D.Double(-RADIUS_ARCSEC, -RADIUS_ARCSEC, RADIUS_ARCSEC*2, RADIUS_ARCSEC*2);
 
-    // Width of CWFS probe arm in arcsec
-//    private static final double PROBE_ARM_WIDTH = 12.95;
-    private static final double PROBE_ARM_WIDTH = 10.0; // see REL-160
-
-    // Probe arm end, as distance in arcsec from the CWFS guide star
-    private static final double PROBE_ARM_END = 8.9;
-
-
     /**
      * Returns a Shape that defines the AO port in arcsecs, centered at
      * <code>(0,0)</code>.
      */
     private static final Shape AO_PORT_SHAPE = new Ellipse2D.Double(AO_PORT.getX(), AO_PORT.getY(), AO_PORT.getWidth(), AO_PORT.getHeight());
```

Comment:
I wonder what the difference between the `AO_PORT_SHAPE` and `AO_PORT` is?  This is also now the "probe range" as I understand it.